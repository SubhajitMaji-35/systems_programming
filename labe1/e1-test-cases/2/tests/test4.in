7 2 9 4
1 8 3 6
5 0 2 7
9 3 1 5
