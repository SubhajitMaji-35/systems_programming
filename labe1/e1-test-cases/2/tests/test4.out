7 1 5 9 
2 8 0 3 
9 3 2 1 
4 6 7 5 
